ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-rollback-stable.yaml

```diff
@@ -20,7 +20,7 @@ releases:
     release:
       architecture: amd64
       channel: candidate
-      version: 4.18.0-rc.0
+      version: 4.18.1
```

Comment:
I'll wait for a peer review to ack this change.

Reply:
1. https://github.com/openshift/release/pull/55692#discussion_r1732063254
`The rule is that we always use the oldest stable version as the oldest rollback version in the testing.`

2. https://redhat-internal.slack.com/archives/CF8SMALS1/p1733193009943209?thread_ts=1733192438.254849&cid=CF8SMALS1

3. https://redhat-internal.slack.com/archives/CH76YSYSC/p1740062966916229
4.18.1 will be final ga.

So the first stable build can be used in rollback